unit-tests/utilities/time/test-waiting-on.cpp

```diff
@@ -92,9 +94,7 @@ TEST_CASE( "Not invoked but still notified" )
     
     // Add something we'll be waiting on
     utilities::time::waiting_on< bool > invoked( false );
-    dispatcher->push( [invoked_in_thread = invoked.in_thread()]() {
-        invoked_in_thread.signal( true );
-    } );
+    dispatcher->push( [invoked_in_thread = invoked.in_thread()]() { invoked_in_thread.signal( true ); } );
```

Comment:
This is because I tried making some changes to remove a warning but it caused the test to fail. So I reverted but left the formatting according to the clang-format